### DISCUSSION TITLE
Do you also use Locus at the same time?

### DISCUSSION THREAD

**Discussion by cnchip** [osmandapp/OsmAnd#16333] [General]
If used, why ?
Aha, this question can be discussed ?
These two apps are the best apps I know of, and I'm curious if there are even better ones on this planet.
I believe this is a place for users who love OsmAnd and look forward to seeing what you thinking.
Thanks.

**Comment by mariush444** [comment]
I use oruxmap because wms, wmts and other online services are implemented in much better way.
Additionally orux can use calibrated maps like ozi, geotiff, geopdf ... and other standards like rmap, garmin ...
Orux can use brouter in off line mode, it is great too for me.

**Comment by readonly24** [comment]
Yes, I do, because of its very good point and track handling.
